### ISSUE TITLE
Add a `topic-*` label for common (public) standard library module 

### ISSUE THREAD

**Issue by picnixz** [python/core-workflow#547]
> Personally I'd probably favor a label for virtually any standard library module that gets a significant issue volume; I don't think the cost of a new label is very high. But others may disagree.

_Originally posted by @JelleZijlstra in https://github.com/python/core-workflow/issues/545#issuecomment-2286848045_
            
When this was first suggested, I was like "ok why not?" and was a +0. But some time has passed again and I really think that it's worth it. It allows triagers to categorize issues way more simply when a title is cryptic and draw attention to the involved component. 

I'm pretty sure that some labels will never be used or only used once because the module is forgotten (e.g., the `graphlib` or the `netrc` module). But I think we can make it once and for all. It's not really an issue for triagers to search through the labels and it would also let me feel a bit less sad when I can't find the label and can just put `stdlib` on it. 

There are some modules that could be gathered under similar labels, e.g., `annotationlib` could be `topic-typing` unless Jelle wants something more precise for this specific typing part.

On the other hand, what we could do as a first step, is to add labels for the existing projects. The rationale behind is that labels are visible on the issue list while projects are only visible if you open the issue (or am I mistaken because of refined github?)

I'll write a script to actually create the labels that need to be created (maybe not today) but I'd like to raise this question in a more visible way.

**Comment by hugovk** [comment]
Jelle suggested for "any standard library module _that gets a significant issue volume_" (emphasis added).

I agree with this.

But I'm not sure we'd want to add hundreds of labels for all the modules:

```pycon
>>> import sys
>>> len(sys.stdlib_module_names)
288
```

It would increase the task of maintaining the labels: adding new ones, deciding what to do with removed modules -- remove or merge, and so on.

**Comment by picnixz** [comment]
> But I'm not sure we'd want to add hundreds of labels for all the modules:

Yes, it was more than what I expected :') But there are some that can require a topic I think, at least those that have projects! (like `topic-unittest` or `topic-logging` does not exist!)
